<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 85, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QPushButton" name="BtnToggle">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/image/imagens/menu-bar (1).png</normaloff>:/image/imagens/menu-bar (1).png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="LeftMenu">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>210</width>
        <height>450</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QToolBox" name="Toggle">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>201</width>
           <height>450</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="Home">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>188</width>
            <height>371</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Home</string>
          </attribute>
          <widget class="QPushButton" name="Btn_Vendas">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>224</y>
             <width>200</width>
             <height>50</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
   border-radius: 8px
}

QPushButton:hover{
  background-color:rgb(50,50,50);
  color:rgb(200,200,200)
}

QPushButton:pressed{
  background-color:rgb(35,35,35);
  color:rgb(200,200,200)
}</string>
           </property>
           <property name="text">
            <string>Vendas</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/image/imagens/Cash-register.32.png</normaloff>:/image/imagens/Cash-register.32.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="Btn_Usuarios">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>280</y>
             <width>200</width>
             <height>50</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
   border-radius: 8px
}

QPushButton:hover{
  background-color:rgb(50,50,50);
  color:rgb(200,200,200)
}

QPushButton:pressed{
  background-color:rgb(35,35,35);
  color:rgb(200,200,200)
}</string>
           </property>
           <property name="text">
            <string>Usuarios</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/image/imagens/cade.png</normaloff>:/image/imagens/cade.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="Btn_Produtos">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>168</y>
             <width>200</width>
             <height>50</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
   border-radius: 8px
}

QPushButton:hover{
  background-color:rgb(50,50,50);
  color:rgb(200,200,200)
}

QPushButton:pressed{
  background-color:rgb(35,35,35);
  color:rgb(200,200,200)
}</string>
           </property>
           <property name="text">
            <string>Produtos</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/image/imagens/Produtos.png</normaloff>:/image/imagens/Produtos.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="Btn_Estoque">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>112</y>
             <width>200</width>
             <height>50</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
   border-radius: 8px
}

QPushButton:hover{
  background-color:rgb(50,50,50);
  color:rgb(200,200,200)
}

QPushButton:pressed{
  background-color:rgb(35,35,35);
  color:rgb(200,200,200)
}</string>
           </property>
           <property name="text">
            <string>Estoque</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/image/imagens/editar (1).png</normaloff>:/image/imagens/editar (1).png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="Btn_Compras">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>56</y>
             <width>200</width>
             <height>50</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
   border-radius: 8px
}

QPushButton:hover{
  background-color:rgb(50,50,50);
  color:rgb(200,200,200)
}

QPushButton:pressed{
  background-color:rgb(35,35,35);
  color:rgb(200,200,200)
}</string>
           </property>
           <property name="text">
            <string>Compras</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/image/imagens/vendas.png</normaloff>:/image/imagens/vendas.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="Btn_Clientes">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>200</width>
             <height>50</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
   border-radius: 8px
}

QPushButton:hover{
  background-color:rgb(50,50,50);
  color:rgb(200,200,200)
}

QPushButton:pressed{
  background-color:rgb(35,35,35);
  color:rgb(200,200,200)
}</string>
           </property>
           <property name="text">
            <string>Clientes</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/image/imagens/users.png</normaloff>:/image/imagens/users.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="Ajuda">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>188</width>
            <height>371</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Ajuda</string>
          </attribute>
          <widget class="QPushButton" name="Btn_Contatos">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>200</width>
             <height>50</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
   border-radius: 8px
}

QPushButton:hover{
  background-color:rgb(50,50,50);
  color:rgb(200,200,200)
}

QPushButton:pressed{
  background-color:rgb(35,35,35);
  color:rgb(200,200,200)
}</string>
           </property>
           <property name="text">
            <string>Contatos</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/image/imagens/whatsapp-logo.png</normaloff>:/image/imagens/whatsapp-logo.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="Btn_Sobre">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>60</y>
             <width>200</width>
             <height>50</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
   border-radius: 8px
}

QPushButton:hover{
  background-color:rgb(50,50,50);
  color:rgb(200,200,200)
}

QPushButton:pressed{
  background-color:rgb(35,35,35);
  color:rgb(200,200,200)
}</string>
           </property>
           <property name="text">
            <string>Sobre</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/image/imagens/Info.png</normaloff>:/image/imagens/Info.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>43</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Kdu 2023 &lt;img src=&quot;:/image/imagens/d.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="L_Logado">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivos">
    <property name="title">
     <string>Arquivos</string>
    </property>
    <widget class="QMenu" name="menuAbrir">
     <property name="title">
      <string>Abrir</string>
     </property>
     <addaction name="actionClientes"/>
     <addaction name="actionCompras"/>
     <addaction name="actionEstoque"/>
     <addaction name="actionProdutos"/>
     <addaction name="actionVendas"/>
     <addaction name="actionUsuarios"/>
    </widget>
    <addaction name="menuAbrir"/>
    <addaction name="actionSalvar"/>
    <addaction name="actionFechar"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionContatos"/>
    <addaction name="actionSobre"/>
   </widget>
   <addaction name="menuArquivos"/>
   <addaction name="menuAjuda"/>
  </widget>
  <action name="actionSalvar">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/image/imagens/Save.32.png</normaloff>:/image/imagens/Save.32.png</iconset>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionFechar">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/image/imagens/close.png</normaloff>:/image/imagens/close.png</iconset>
   </property>
   <property name="text">
    <string>Fechar</string>
   </property>
  </action>
  <action name="actionClientes">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/image/imagens/users.png</normaloff>:/image/imagens/users.png</iconset>
   </property>
   <property name="text">
    <string>Clientes</string>
   </property>
   <property name="toolTip">
    <string>Cadastro de Clientes</string>
   </property>
  </action>
  <action name="actionCompras">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/image/imagens/S_Compras.png</normaloff>:/image/imagens/S_Compras.png</iconset>
   </property>
   <property name="text">
    <string>Compras</string>
   </property>
   <property name="toolTip">
    <string>Ca dastro de Compras</string>
   </property>
  </action>
  <action name="actionEstoque">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/image/imagens/editar (1).png</normaloff>:/image/imagens/editar (1).png</iconset>
   </property>
   <property name="text">
    <string>Estoque</string>
   </property>
   <property name="toolTip">
    <string>Controle de Estoque</string>
   </property>
  </action>
  <action name="actionProdutos">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/image/imagens/Produtos.png</normaloff>:/image/imagens/Produtos.png</iconset>
   </property>
   <property name="text">
    <string>Produtos</string>
   </property>
   <property name="toolTip">
    <string>Tela de Produtos</string>
   </property>
  </action>
  <action name="actionVendas">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/image/imagens/Cash-register.32.png</normaloff>:/image/imagens/Cash-register.32.png</iconset>
   </property>
   <property name="text">
    <string>Vendas</string>
   </property>
   <property name="toolTip">
    <string>Cadastro de Vendas</string>
   </property>
  </action>
  <action name="actionContatos">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/image/imagens/whatsapp-logo.png</normaloff>:/image/imagens/whatsapp-logo.png</iconset>
   </property>
   <property name="text">
    <string>Contatos</string>
   </property>
  </action>
  <action name="actionSobre">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/image/imagens/Info.png</normaloff>:/image/imagens/Info.png</iconset>
   </property>
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionUsuarios">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/image/imagens/cadastro.png</normaloff>:/image/imagens/cadastro.png</iconset>
   </property>
   <property name="text">
    <string>Usuarios</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
